thentos-core/exec/ImportAulaUsers.hs

```diff
@@ -0,0 +1,49 @@
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+
+module Main where
```

Comment:
The file name suggests this should probably go to the aula repo.  But you add a FIXME in if you don't want to do it.